### DISCUSSION TITLE
Low nodejs version

### DISCUSSION THREAD

**Discussion by wojiaoxiaomayun** [unocss/unocss#2201] [Ideas]
I used low nodejs version,I want to use unocss,but it's very bad.Must be version 16 to support.

**Comment by StarLederer** [comment]
Considering that UnoCSS works in a literal browser I think you are targeting something that is too outdated. NodeJS grants every even version long term support and not even Node 16 is the latest one anymore. Could you please specify why you can't use Node 16 or higher in your case?

**Reply by wojiaoxiaomayun** [reply]
Yes, the high version is indeed good, but I am currently unable to upgrade due to the low version of Node used by the project, and there are too many project maintenance staff, I have imitated unocss to write a set of real-time parsing code, which may be worse than unocss performance, but it is completely enough for me. Real-time parsing of DOM conversion styles through custom directives. Thank you for your answer
